### PR TITLE
Try move auth

### PR DESCRIPTION
**Pull request by theRealImy**
This is the PR for the move auth from solid-ui to solid-logic. depends on solid-logic and pane-registry

### CODE REVIEW

**Comment by bourgeoa** [comment]
@theRealImy do you know why rests are waiting ?
Is there a CI problem?

**Comment by theRealImy** [comment]
> @theRealImy do you know why rests are waiting ? Is there a CI problem?

@bourgeoa not sure... Basically, it is not running them at all. 🤨 Maybe if I push smth again it might jump start it. I am waiting on some reviews.

**Comment by timbl** [comment]
"npm ERR! 403 403 Forbidden - PUT https://registry.npmjs.org/solid-ui - You cannot publish over the previously published versions: 2.4.16-48f3ebed."
Trying to re-publish the same version?  A CI script out of sync with itself?

File: README.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 User Interface widgets and utilities for Solid
 
 These are HTML5 widgets which connect to a solid store. Building blocks for solid-based apps.
+Vanilla JS.  Includes large widgets like chat, table, matrix; form fields, and small widgets.
```
Comment (TallTed) [inline README.md:8]:
```suggestion
Vanilla JS.  Includes large widgets like chat, table, matrix, form fields, and small widgets.
```

**Pull request merged**